ibdgc.datacommons.io/etlMapping.yaml

```diff
@@ -2,7 +2,7 @@ mappings:
   - name: ibdgc_etl
     doc_type: case
     type: aggregator
-    root: case
+    root: particpant
     props:
```

Comment:
In the dictionary looks like it’s “case”, the front end text just says participant 
So need to change to case

Reply:
Check